### ISSUE TITLE
ubuntu scc vs universe tools

### ISSUE THREAD

**Issue by jcayouette** [SUSE/doc-susemanager#592]
We need to split the ubuntu setup section into an ifeval. 

* Ubuntu for SCC/suma with the client tools (only available via SCC)
* Ubuntu for Uyuni opensource and how to add universe repositories

See the installation guide nav ifeval as an example:

```
ifeval::[{suma-content} == true]
*** xref:clients-ubuntu-scc.adoc[Ubuntu Clients SCC]
endif::[]
*** xref:clients-ubuntu-open.adoc[Ubuntu Clients Universe]
```

* For Uyuni Ubuntu opensource create a new section specific to universe repos and how to add them
* For SUMA create a section that only displays similar to the above code example when docs build for SUMA.

**Comment by juliogonzalez** [comment]
Ping @Loquacity as this is quite urgent.

I talked to @paususe and we should add at least another channel for both SUSE Manager and Uyuni for the security repositories (backports is optional, so I'd say: let's mention it but not require it)

**Comment by Loquacity** [comment]
Yeah, we just need an ifeval for this section: @jcayouette go ahead and do that, you understand the content better than I do at this point.

I think we need a wider discussion on how to handle Uyuni/SUMA variations from here on in, though. The model we are using now is going to break down *very* quickly.

**Comment by keichwa** [comment]
On 2019-07-26 02:46, Lana Brindley wrote:

> I think we need a wider discussion on how to handle Uyuni/SUMA
> variations from here on in, though. The model we are using now is
> going to break down *very* quickly.

:+100:

-- 
Karl Eichwalder

**Comment by jcayouette** [comment]
I think the real issue is knowing what we have available for both outputs and what is missing. We should perform a total analysis of docs and make note of content that is relevant for Uyuni and for SUMA. 

For example just briefly scanning the Uyuni PDF's last night I came across our SCC section of the installation guide.  This does not apply to open source in most cases. (Granted someone could use SCC if they chose too.)

Assumed User Perspectives

- Enterprise SCC
- Open Source no SCC
- Mixed SCC and Open

We need to take a deep dive into how we will organize this type of content, and come up with an appropriate solution that is straightforward and maintainable.

The attributes placed into our site.yml files are added to the antora catalog. This means they are available globally. I wonder if we could set a few ifeval statements there. Then apply the attribute directly to the documents themselves. This would clean up our nav lists :)

**Comment by keichwa** [comment]
Joseph Cayouette <notifications@github.com> writes:

> We need to take a deep dive into how we will organize this type of
> content, and come up with an appropriate solution that is
> straightforward and maintainable.

I'm all for it, but ...

> This means they are available globally. I wonder if we could
> set a few ifeval statements there. Then apply the attribute directly to
> the documents themselves.

ifdef things are neither straightforward nor maintainable.

Straightforward and maintainable for me is: Write a minimal "Uyuni
Installation and User Guide".  To that guide attach a "For more
information about ..., see ..." list that references the SUMA
documentation.  Of course, with a caveat at the beginning of this list;
there we ask the Uyuni user to read the referenced SUMA documenation
just for inspiration, etc.

We then will enhance the Uyuni documentation if time permits or if
priorities are shifted.

Yes, this is work, but doable.  And it will not make existing docs even
more chaotic.

> This would clean up our nav lists :)

What shall I say...

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)
